src/hedgehog.js

```diff
@@ -42,7 +42,8 @@ class Hedgehog {
       const authData = {
         iv: ivHex,
         cipherText: cipherTextHex,
-        lookupKey: lookupKey
+        lookupKey: lookupKey,
+        walletAddress
```

Comment:
@raymondjacobson @vicky-g i don't think we should do this. the reason we don't associate auth information with wallet address is because it introduces the possibility to leak the correlation between auth data and a wallet. would it be possible to do this at the user route?

Reply:
yup, can do that instead. @raymondjacobson any different opinions?

Reply:
that's fine w/ me. ignore my comment on the other PR about not doing it in the user route--let's just do it all there. FWIW though i don't think it materially is all that different @dmanjunath -- it would be still almost as easy to snoop someone's request data and find the two requests and associate that way, so long as we don't store the wallet address alongside the auth data.